packages/core/src/dialects/sqlite/query-generator.js

```diff
@@ -304,15 +304,12 @@ export class SqliteQueryGenerator extends SqliteQueryGeneratorTypeScript {
 
     attributes = this.attributesToSQL(attributes);
 
-    let backupTableName;
-    if (typeof tableName === 'object') {
-      backupTableName = {
-        tableName: `${tableName.tableName}_backup`,
-        schema: tableName.schema,
-      };
-    } else {
-      backupTableName = `${tableName}_backup`;
-    }
+    const table = this.extractTableDetails(tableName);
+
+    const backupTableName = {
+      tableName: `${table.tableName}_backup`,
+      schema: table.schema,
```

Comment:
Not something for this PR but this is unsafe: This table could already exist

Reply:
Added a TODO for it